lib/3scale_toolbox/commands/import_command/openapi/create_activedocs_step.rb

```diff
@@ -11,7 +11,8 @@ def call
               system_name: activedocs_system_name,
               service_id: service.id,
               body: resource.to_json,
```

Comment:
When touching this you might want to use JSON.pretty_generate, so the JSON looks ok to humans.
Because you are already decoding it, so the original formatting is gone.

Reply:
but this json serialization will not be consumed by humans, just 3scale api that requires json serialization.

Reply:
It is, it appears in the UI when you edit the spec.

Reply:
I think there are 3 options (ordered by their correctness in my eyes)

1. upload the original bytes and keep the original whitespace
2. upload a pretty printed copy, so humans can actually read that file in the UI 
3. upload a compacted copy for computers

Reply:
ok, let it be the original bytes then

Reply:
3scale API only accepts JSON serialization format.

So I guess, best option is upload a pretty printed copy

Reply:
I mean, pretty json serialization